### DISCUSSION TITLE
Battery Storedge control

### DISCUSSION THREAD

**Discussion by dlipner** [WillCodeForCats/solaredge-modbus-multi#187] [Q&A]
Hi, is the control of the SE battery working?
I seem to be unable to find a way to force-charge the battery from the grid with this integration. I am able to access all other values/sensors.

**Comment by mpredfearn** [comment]
Yes, it should work, if you set the storage control mode to "remote control" you should then be able to change the default mode as you desire.

**Comment by seatrope** [comment]
@mpredfearn how do you change the storage control mode to "remote control"? is this in the integration or via Setapp? thank you!

**Reply by mpredfearn** [reply]
Use the `select.solaredge_i1_storage_control_mode` control from this integration.
![Screenshot from 2024-07-12 15-22-37-2](https://github.com/user-attachments/assets/f694ca2c-2e94-4060-859e-12805395ee9a)

**Comment by seatrope** [comment]
@mpredfearn  Thank you Matt! I can't get the Power Control part working, as per my other post here. so that's why... I was wondering if the Remote Control option was why I couldn't get Power Control working... chicken or egg.
